projects/plugins/jetpack/extensions/blocks/map/test/validate.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Internal dependencies
+ */
+import { name, settings } from '../';
+import runBlockFixtureTests from '../../../shared/test/block-fixtures';
+
+/**
+ * 👀 Update this array of blocks to contain the name and settings for all blocks
```

Comment:
Tiny nit: we can remove this comment block since this block doesn't need other blocks 🙂

Reply:
Removed :+1: